storage/core/src/main/java/com/hortonworks/streamline/storage/impl/jdbc/provider/mysql/query/MySqlQuery.java

```diff
@@ -20,13 +20,15 @@
 import com.hortonworks.streamline.storage.impl.jdbc.provider.sql.query.AbstractSqlQuery;
 
 public class MySqlQuery extends AbstractSqlQuery {
+    private final String theSql;
```

Comment:
right, i had renamed sql->theSql to identify the places where the variable was used earlier and forgot to rename.